app/src/main/res/layout/activity_event.xml

```diff
@@ -519,7 +553,7 @@
                 android:id="@+id/event_availability"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_below="@+id/event_status_divider"
+                android:layout_below="@+id/event_access_level_divider"
```

Comment:
Just FYI, this breaks the layout when 'Store locally only' is selected. I'm working on it.

<img src='https://github.com/user-attachments/assets/ddc07276-3357-4fab-b538-10274653cacc' width=200 />